week6/08-prokhodnoi-ball/task.py

```diff
@@ -0,0 +1,54 @@
+class Student:
+    name = ""
+    first = 0
+    second = 0
+    third = 0
+    sum = 0
+
+
+def main():
+    with open("input.txt", encoding="utf-8") as input_file, open(
+            "output.txt", "w", encoding="utf-8") as output_file:
+        k = int(input_file.readline())
+        input_lines = input_file.readlines()
+        print(get_min_score(input_lines, k), file=output_file)
+
+
+def count_by_key(input_list, pattern, key=lambda x: x):
+    res = 0
+    for el in input_list:
+        if key(el) == pattern:
+            res += 1
+    return res
+
+
+def get_min_score(input_lines, k):
+    students = []
+    for line in input_lines:
+        student = Student()
```

Comment:
extract string to class method parse_student

Reply:
done